scripts/codecov.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SHA="-C $SEMAPHORE_GIT_SHA"
+BRANCH="-B $SEMAPHORE_GIT_BRANCH"
+PR=
+if [ "x$SEMAPHORE_GIT_PR_NUMBER" != "x" ]; then
+    PR="-P $SEMAPHORE_GIT_PR_NUMBER"
+    SHA="-C $SEMAPHORE_GIT_PR_SHA"
+    BRANCH="-B $SEMAPHORE_GIT_PR_BRANCH"
+fi
+BUILD="-b $SEMAPHORE_JOB_ID"
+NAME="-n '$SEMAPHORE_JOB_NAME'"
+
+# Are we in a PR context? The variables are wrong if so.
+if [ "x$SEMAPHORE_GIT_REF_TYPE" == "xpull-request" ]; then
```

Comment:
```suggestion
if [[ "$SEMAPHORE_GIT_REF_TYPE" == "pull-request" ]]; then
```

bash with the `[[`-style test doesn't need the prefix!